### PR TITLE
[SMALLFIX] Fix "Allocation after freeing space" log message

### PR DESCRIPTION
**Pull request by bradyoo**
dirView has no toString implementation. So it is certainly not what you want.

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java [hunk 1]
```diff
@@ -652,7 +652,10 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options) {
           // Block expansion are forcing the location. We do not want the review's opinion.
           dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
               options.getLocation(), allocatorView.refreshView(), true);
-          LOG.debug("Allocation after freeing space for block expansion: {}", dirView);
+          if (LOG.isDebugEnabled()) {
+            LOG.debug("Allocation after freeing space for block expansion: {}",
```
Comment (gpang) [inline core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java:656]: This should probably be something like "Allocation after freeing space for block expansion. availableBytes: {}". Is the goal of this message to show the available bytes after freeing?

**Review by gpang — CHANGES_REQUESTED** [review]
@bradyoo Thanks! Can you update the PR description with the before and after? It is not clear to me what the old and new message look like.

**Review by jiacheliu3 — APPROVED** [review]
DItto for Gene's comment above. LGTM if the comment is resolved. Thanks!

**Comment by gpang** [comment]
```
+ say '\033[0;31m-->\033[0m No coverage report found.'
+ echo -e '\033[0;31m-->\033[0m No coverage report found.'
--> No coverage report found.
+ say '    Please visit \033[0;36mhttp://docs.codecov.io/docs/supported-languages\033[0m'
+ echo -e '    Please visit \033[0;36mhttp://docs.codecov.io/docs/supported-languages\033[0m'
+ exit 1
    Please visit http://docs.codecov.io/docs/supported-languages
Error: Codecov failed with the following error: The process '/bin/bash' failed with exit code 1
```

**Comment by bradyoo** [comment]
generate code coverage flaked. Code Coverage generation has been flaky. Upload code coverage failed because generating it failed.

**Comment by bradyoo** [comment]
done.

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks!

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Pull request merged**